public/video-ui/src/util/getComposerData.js

```diff
@@ -77,6 +78,19 @@ export function getComposerData(video) {
       belongsTo: 'thumbnail'
     }
   ];
+
+  const isTrainingMode = getStore().getState().config.isTrainingMode;
+
+  // block training content being published
+  const embargoedIndefinately = {
+    name: 'embargoedIndefinitely',
+    value: asBooleanString(true),
+    belongsTo: 'settings'
+  };
```

Comment:
Do we need to add this here? We are already embargoing the article on page creation, does this do anything other than also embargo it on subsequent publications?